### ISSUE TITLE
If write SMS db permission is not disabled, double SMS are in my log

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#365]
```
What steps will reproduce the problem?
1. Disable the write SMS DB permission in app ops
2. Send an SMS via gchat

What is the expected output?
I can see the text message I sent in my text message app

What do you see instead?
Two copies of the SMS are in there, while only one was sent in reality.

Additional informations:
Phone model: Nexus 4, vanir
```

Original issue reported on code.google.com by `catlvr...@gmail.com` on 16 Aug 2014 at 3:40

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 1652e3663335.
```

Original comment by `yako...@gmail.com` on 31 Aug 2014 at 1:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the last beta
```

Original comment by `Florent....@gmail.com` on 31 Aug 2014 at 6:32
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
